packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -327,10 +327,18 @@ def expected_service_running(
     with log_context(logging.INFO, msg="Waiting for node to run") as ctx:
         waiter = SocketIONodeProgressCompleteWaiter(node_id=node_id, logger=ctx.logger)
         service_running = ServiceRunning(iframe_locator=None)
-        with websocket.expect_event("framereceived", waiter, timeout=timeout):
-            if press_start_button:
-                _trigger_service_start(page, node_id)
-            yield service_running
+
+        try:
+            with websocket.expect_event("framereceived", waiter, timeout=timeout):
+                if press_start_button:
+                    _trigger_service_start(page, node_id)
+        except Exception:
+            ctx.logger.warning(
+                "⚠️ Progress bar didn't recieve 100 percent: %s ⚠️",
```

Comment:
"receive"

Reply:
Alsp please add a reference to this PR (e.g. url) to understand in the future what lead us to ignore this check